esrally/utils/sysstats.py

```diff
@@ -45,7 +45,8 @@ def cpu_model():
     """
     :return: The CPU model name.
     """
-    return cpuinfo.get_cpu_info()["brand"] if cpuinfo_available else "Unknown"
+    cpu_info = cpuinfo.get_cpu_info()
```

Comment:
The idea behind this single line statement was that we are lenient whether the `cpuinfo` package is supported on the target platform (see comment at the top in `sysstats`). If we execute `cpuinfo.get_cpu_info()` unconditionally we'd raise an error on those platforms.